monero-outreach-docs/translations/fr/website-sections/your_privacy_fr.md

```diff
@@ -0,0 +1,25 @@
+# Votre Confidentialité
+
+**"L'oiseau en cage chante d'un chant effrayé, de choses inconnues mais toujours désirées, et son chant est perçu sur les collines lointaines, car l'oiseau en cage chante à propos de la liberté" - Maya Angelou - 4th of January, 2021**
+
+Avec plusieurs politiques de confidentialité écrites dans le but de confondre et décourager la lecture, nous sommes reconnaissants d'avoir l'opportunité d'aborder votre vie privée, et notre responsabilité envers elle. Il s'agit de Monero, après tout, et on se doit de faire nous-mêmes ce que nous prêchons.
+
+TLDR: Notre site conserve le moins de données possibles, pour le moins longtemps possible, de la manière la plus sécure possible. Nous ne partagerons pas vos données avec qui que ce soit pour du profit, mais si ça se retrouve sur un serveur, considérez-le du domaine publique.
+
+**Tiers**
+Les services web fournis par des tiers diminuent votre confidentialité, donc nous les évitons comme la peste. La seule exception est que nous faisons appel aux services de Mapbox, Inc. et Mapbox International, Inc. pour fournir nos tuiles de carte. Nos cartes sont désactivées par défaut; si vous les activez, Mapbox aurait vos métadonnées identifiantes venant de nos pages qui contiennent des cartes. Nous étiquetterons clairement chaque instance sur notre site où des tiers verront vos métadonnées et nous vous donnerons la chance de les éviter. Si jamais nous faisons appel à un autre service tiers, nous en ferons mention ici.
+
+**Cookies**
+Nous avons un cookie à option d'adhésion qui génère un identifiant. Cet identifiant enregistre vos préférences sur notre serveur. C'est optionnel, et l'intention est simplement de faciliter l'utilisation de notre site.
+
+**Suivi/Divulgation**
+Notre site suit les visites de page et les enregistre sur notre serveur. Nous n'enregistrons pas votre adresse IP; nous nettoyons votre string d'agent utilisateur et nous n'enregistrons que votre type de fureteur internet et la taille de votre écran. Vos données sont épurées et nous obtenons des statistiques pour mieux orienter nos efforts.
```

Comment:
nous nettoyons votre string d'agent utilisateur >>> nous nettoyons la chaine de caractères de votre agent utilisateur  
Reason: 'string' was not translated


fureteur >>> navigateur 
Reason: suggested because 'fureteur' is a term that might not be familiar to all French native speaker, but (I think) 'navigateur' could be more widely spread (or is just be because of my regional French?)

Reply:
For String - I didn't translate it because it's super common for techies to use that term, even in French. I've integrated your recommendation nonetheless.

For fureteur, I've integrated as in the other texts.